force-app/main/default/lwc/formulaBuilder/formulaBuilder.css

```diff
@@ -0,0 +1,29 @@
+.container {
```

Comment:
Instead of explicitly providing styles here, I'd like to use slds directives per https://www.lightningdesignsystem.com/design-tokens/ 
For example, instead of explicitly going with white, use $color-background-alt, for radius, use the values at https://www.lightningdesignsystem.com/design-tokens/#category-radius , etc.

Reply:
@alexed1 Updated